### PR TITLE
TR-642, TR-666 add request/response example

### PR DESCRIPTION
**Pull request by ext-attelaine**

### CODE REVIEW

**Review by malinenesko — APPROVED** [review]

File: examples/report_update_request_response_example.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+REQUEST:
+
+POST https://[ACCOUNT_REGISTER] HTTP/1.1
+Accept-Encoding: gzip,deflate
+Content-Type: application/json
+Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImlhdCI6MTU4OTI2MzEyNn0.eyJhdWQiOiJhY2NvdW50UmVnaXN0ZXIiLCJzdWIiOiI4NDg4ODI5LTYifQ.mF3sQBGpbxy-zzqjw-vz30oMxizjDNpHog-wa1VWKiy8I_TQ_NVCusm6xpkqOqLuew8xprtQTRz_xB3iIT820X-ouY93trxv6HlWADBDGLzNVVwI7P9rb9aJfTl0VOUJSfHqMyyoKKCi2PWFEokNDhiLD0spYKGosbWz-mB_rKB1pv3PxwFK7SnpLqF62C2qjyQiwn2EQgk7FAd-iJUwxdYmIsRUYxI1EILexgEh27v0D-IC0UMXcAeoAJbqoKA1WY1MioBgfTL7MXDxDuttC0hP1ixrDqZxOhejbGfBjbU6icjrrlrvej6NugQgFcG3j5XTahYOtRZxsF1Y7Eqt4w
```
Comment (ext-alexpeltonen) [inline examples/report_update_request_response_example.txt]: "Bearer" https://tools.ietf.org/html/rfc6750#section-6.1.1

File: examples/report_update_request_response_example.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+REQUEST:
```
Comment (ext-alexpeltonen) [inline examples/report_update_request_response_example.txt]: Should we put request and response in different files, without comments, verbatim?

**Review by ext-alexpeltonen — CHANGES_REQUESTED** [review]
Bearer scheme name needs to be fixed

**Review by PaavoKaltiainen — APPROVED** [review]

File: examples/request_response_example/request.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...WI1Ijp7ImlkIjoiRkktMzQ1MDIwMDIyNCIsInN0YXJ0RGF0ZSI6IjIwMTUtMDItMTEiLCJlbmREYXRlIjoiMjAxOS0wNy0wMyIsInJvbGVzIjpbeyJsZWdhbFBlcnNvbklkIjoiODI5MTE3MzktNmY5Ny00MzllLWI1NjEtMjJmYWMzMjdiNzc0Iiwic3RhcnREYXRlIjoiMjAxNS0wMi0xMSIsImVuZERhdGUiOiIyMDE5LTA3LTAzIiwidHlwZSI6Im93bmVyIn0seyJsZWdhbFBlcnNvbklkIjoiM2IxNTNkNDQtNGNhYS00ZmViLWJkZDEtYTkxYjZiMTc1OWNlIiwic3RhcnREYXRlIjoiMjAxNS0wMi0xMSIsImVuZERhdGUiOiIyMDE5LTA3LTAzIiwidHlwZSI6ImFjY2VzcyJ9LHsibGVnYWxQZXJzb25JZCI6ImViYWIwYTQ3LTAwYTctNGQ1Ni1hYTYzLTAyODEwMmQ3YTMyOSIsInN0YXJ0RGF0ZSI6IjIwMTUtMDktMzAiLCJlbmREYXRlIjoiMjAxOS0wNy0wMyIsInR5cGUiOiJhY2Nlc3MifV19LCI0YzNlYjUyNC0wYzA2LTQ1NzUtOWNjMi1mMWEyNDhmNzZiYjEiOnsiaWQiOiJGSS03NzIwNzg2NzYiLCJzdGFydERhdGUiOiIyMDE2LTAyLTExIiwicm9sZXMiOlt7ImxlZ2FsUGVyc29uSWQiOiIyNzlhZWFmYy01MjhlLTRmODEtYjc5Yy1lY2M4YjBhYWZkNGMiLCJzdGFydERhdGUiOiIyMDE2LTEyLTMwIiwidHlwZSI6Im93bmVyIn1dfX19fQ.TP2EHvt5If-GbgzRc90F7rdBGr8aPg5pkAuVTYxJyx-hpH_yp83at73xXoFh9FyZEcNwyQwYjOh4zKDdB1ntl8Cr1MrTpl21aIVmiKf5WgqlCXc0fFJp8yi8ZwURiEEA8ZJokJn4QLyO8qVfETIlEHsqhTLD7169H_XXcf5Nfy_tZvBOwr17jWpxiYW8EN9Zr945oV8vX8J98ewPPISiZevB3NbIAvL6m1PP62djT9c3BkB0kj897Ta5ZY_vjKBRsRko6hVRSavnJuGSSdy_-o-DUMO4Hhj3YIC685dJKiTFOJpXiP-w4EEYNwp5LvEvkEXpCP6LhVRPqYOMixkZtA
```
Comment (ext-alexpeltonen) [inline examples/request_response_example/request.txt]: Replace "Request body:" with empty line

**Review by ext-alexpeltonen — CHANGES_REQUESTED** [review]

**Pull request merged**